docs/getting-started-guides/centos/centos_manual_config.md

```diff
@@ -29,10 +29,10 @@ Hosts:
 Please replace host IP with your environment.
 
 ```conf
-centos-master = 192.168.121.9
-centos-minion-1 = 192.168.121.65
-centos-minion-2 = 192.168.121.66
-centos-minion-3 = 192.168.121.67
+192.168.121.9 centos-master
+192.168.121.65 centos-minion-1
+192.168.121.66 centos-minion-2
+192.168.121.67 centos-minion-3 
```

Comment:
this is only a list for reference later in the article. It is not an artifact for use.
On line 58 there is a command for you to generate the /etc/hosts file.
So I don't agree to this change.